src/components/TaskCheckbox/TaskCheckbox.tsx

```diff
@@ -18,44 +23,53 @@ const useStyles = makeStyles({
   line: {
     width: '100%',
     position: 'absolute',
-    height: '0.1rem',
-    backgroundColor: 'rgba(0, 0, 0, 0.28)',
-    top: '1.2rem',
+    zIndex: -1,
+    height: `${LINE_THROUGH_HEIGHT}rem`,
+    backgroundColor: colors.custom.lightText,
+    top: `calc(50% - ${LINE_THROUGH_HEIGHT / 2}rem)`,
```

Comment:
@Tonik1204  can you explain why those modifications were needed?

Reply:
Sure @lucianojss. We need this be 100% sure that `line` element always will be in the middle - to have the propper cross out effect. In case of previous solution `top: '1.2rem'` - that's not solid enough.